pandas/io/parsers.py

```diff
@@ -825,6 +825,22 @@ def _validate_usecols_arg(usecols):
     return usecols
 
 
+def _validate_parse_dates_arg(parse_dates):
```

Comment:
you are adding new code but not checking for existing code, e.g. see like 298. Further let's go ahead and validate all of the cases (and test whatever is not tested), e.g. list/dict are valid (but need mainly to tests what is not accepted)